etb_format.c

```diff
@@ -78,7 +80,22 @@ int decode_etb_stream(struct stream *etb_stream)
         goto exit_decode_etb_stream;
     }
 
-    for (pkt_idx = 0; pkt_idx < etb_stream->buff_len; pkt_idx += ETB_PACKET_SIZE) {
+    if (unaligned) {
+        for (b = 0; b < etb_stream->buff_len-sizeof(fsync)-1; b++) {
+            if (memcmp(&fsync, &etb_stream->buff[b], sizeof(fsync)) == 0) {
+                ofs = b+sizeof(fsync); break;
+            }
+        }
+
+        if (!ofs) {
+            OUTPUT("No FSYNC found.");
```

Comment:
Please use the full name (rather than the abbreviation) in the error message showing to users.

FSYNC -> frame synchronization packet

Reply:
Fixed in 31443f0.